data/json/items/comestibles.json

```diff
@@ -4324,44 +4324,355 @@
+    "spoils_in": 7200,
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": 1,
+    "calories": 80,
+    "description": "A handful of nuts from an almond tree, their shells have been removed.",
+    "price": 100,
+    "material": "nut",
+    "volume": 1,
+    "flags": [ "EDIBLE_FROZEN" ],
+    "charges": 2,
+    "fun": 2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "almond_roasted",
+    "name": "handful of roasted almonds",
+    "name_plural": "handfuls of roasted almonds",
+    "weight": 32,
+    "color": "brown",
+    "spoils_in": 7200,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": 2,
+    "calories": 120,
+    "description": "A handful of roasted nuts from an almond tree.",
+    "price": 150,
+    "material": "nut",
+    "volume": 1,
+    "flags": [ "EATEN_HOT", "EDIBLE_FROZEN" ],
+    "fun": 3
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cashews",
+    "name": "cashews",
+    "name_plural": "cashews",
+    "weight": 35,
+    "color": "brown",
+    "spoils_in": 5760,
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": 1,
+    "calories": 90,
+    "description": "An imported bag of salty cashews.",
+    "price": 110,
+    "material": "nut",
+    "volume": 1,
+    "flags": [ "EDIBLE_FROZEN" ],
+    "charges": 2,
+    "fun": 3
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pecan_unshelled",
+    "name": "handful of shelled pecans",
+    "name_plural": "handful of shelled pecans",
+    "weight": 36,
+    "color": "brown",
+    "spoils_in": 4320,
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": 1,
+    "calories": 120,
+    "description": "A bag of pecans which are a sub-species of hickory nuts, their shells have been removed.",
+    "price": 100,
+    "material": "nut",
+    "volume": 1,
+    "flags": [ "EDIBLE_FROZEN" ],
+    "charges": 2,
+    "fun": 1
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pecan_roasted",
+    "name": "handful of roasted pecans",
+    "name_plural": "handfuls of roasted pecans",
+    "weight": 36,
+    "color": "brown",
+    "spoils_in": 4320,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": 1,
+    "calories": 120,
+    "description": "A handful of roasted nuts from a pecan tree.",
+    "price": 150,
+    "material": "nut",
+    "volume": 1,
+    "flags": [ "EATEN_HOT", "EDIBLE_FROZEN" ],
+    "fun": 3
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pistachios",
+    "name": "pistachios",
+    "name_plural": "pistachios",
+    "weight": 28,
+    "color": "green",
+    "spoils_in": 2160,
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": 2,
+    "calories": 80,
+    "description": "An imported bag of salty pistachios, their shells have been removed.",
```

Comment:
America is also a world leader in pistachios production.

Reply:
Added pistachio tree.